samples/quickstart.js

```diff
@@ -25,6 +25,8 @@ function main(
 
   // Creates a client
   const storage = new Storage();
+  // Creates a client from the Firebase Admin SDK service key
```

Comment:
Update to `Creates a client from a Google service account key.`

Reply:
Updated.